### PR TITLE
Added option for z0_offset and z1_offset to MakeBundle preset.

### PR DESCRIPTION
**Pull request by ajasja**
Added extra sliders for the MakeBundle sliders.

### CODE REVIEW

File: viewer3d/presets.py [hunk 1]
```diff
@@ -469,12 +471,31 @@ def on_invert_change(change):
...
+      max=3,
+      description="z0_offset",
+      style={"description_width": "initial"},
+      continuous_update=continuous_update,
+    )
+
+    z1_offset = FloatSlider(
+      min=-3,
+      max=3,
+      description="z1_offset",
```
Comment (klimaj) [inline viewer3d/presets.py:539]: Can you also set the `value` attributes in `z0_offset`, `z1_offset` and `delta_omega1`?

**Review by klimaj — APPROVED** [review]
Looks good!

File: viewer3d/presets.py (hunk 1 above)
Comment (ajasja) [reply viewer3d/presets.py:539]: Sure, good idea, for me it was obvious that it's 0, but it's always better to be explict.

**Comment by ajasja** [comment]
@klimaj I'm trying to add a checkbox that would hide/show the sidechains. What's the easiest way to do that? Do I have to delete modules manually?

**Comment by klimaj** [comment]
@ajasja For adding/removing modules in the widget update function, you can directly set `view.set_modules` to overwrite the current list of modules. Otherwise you could `view.clear()` then add the new modules like traditionally. After pose conformational changes and setting the viewer modules, call `view.update_viewer()` if the pose's pointer in memory is the same, otherwise call `view.update_pose(pose)` to update the pose and viewer.

**Pull request merged**